src/map/atcommand.cpp

```diff
@@ -5338,8 +5338,8 @@ ACMD_FUNC(jailfor) {
 	}
 
 	// Added by Coltaro
-	if(pl_sd->sc.data[SC_JAILED] && pl_sd->sc.data[SC_JAILED]->val1 != INT_MAX) { // Update the player's jail time
-		jailtime += pl_sd->sc.data[SC_JAILED]->val1;
+	if(pl_sd->sc.getSCE(SC_JAILED) && pl_sd->sc.getSCE(SC_JAILED)->val1 != INT_MAX) { // Update the player's jail time
+		jailtime += pl_sd->sc.getSCE(SC_JAILED)->val1;
```

Comment:
Stuff like this needs to be addressed with this PR. 3 lookups would be too cost intensive!

Reply:
[74ac183](https://github.com/rathena/rathena/pull/7356/commits/74ac183da74ee1c1df50534f62af1b3b282dfa77) gives an example of a cache of the last-used status, so repeated calls won't cause expensive lookups

it doesn't add much in this case, as it's already a direct access